src/writers/writer.ts

```diff
@@ -151,14 +183,15 @@ export class Writer {
       writingArea: d3.Selection<any>,
       width: number,
       height: number,
+      shearShift: number,
       xAlign: string,
       yAlign: string) {
-
     const lines = text.split("\n");
     const lineHeight = this._measurer.measure().height;
     const yOffset = Writer.YOffsetFactor[yAlign] * (height - lines.length * lineHeight);
     lines.forEach((line: string, i: number) => {
-      this.writeLine(line, writingArea, width, xAlign, (i + 1) * lineHeight + yOffset);
+      const xOffset = (shearShift > 0) ? (i + 1) * shearShift : (i) * shearShift;
```

Comment:
In the first case, we need to add the shift to keep the first line in view, but in the second case, we don't shift the first line.